7853-hmis-hud-2026-breaking-changes

ttoomey opened the pull request:
## _Merging this PR_
- use the squash-merge strategy for PRs targeting a release-X branch

## Description
⚠️ These are breaking API and form schema changes intended for a coordinated, cut-over deployment.

**FY2026 HUD updates across HMIS GraphQL enums and forms**
- New CoC Builds funding source
- Deprecates ESG-CV and HOPWA-CV (breaking enum name changes)
- Revised family critical issues picklists (to `NoYesReasonsForMissingData`)
- Adds V10 mental health consultation fragment to intake (with rule gating)
- Adds `#4_21_sex` demographics fragment
- Adjusts Service form to collect Date Provided for SSVF financial assistance, with additional guidance
- Remove requirement to collect specific disability fields for some VA funders


## Type of change
New feature

## Checklist before requesting review
[//]: # (Remove any items that are not applicable)
- [x] I have performed a self-review of my code
- [x] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [x] I have updated the documentation (or not applicable)
- [ ] I have added spec tests (or not applicable)
- [ ] I have provided testing instructions in this PR or the related issue (or not applicable)

ttoomey committed:
7853-hmis-hud-2026-breaking-changes

ttoomey commented on drivers/hmis/app/graphql/schema.graphql:
```diff
@@ -5293,9 +5298,9 @@ enum FundingSource {
+  (47) HUD: ESG - CV [Deprecated]
   """
-  HUD_ESG_CV
+  HUD_ESG_CV_DEPRECATED
```
adding _DEPRECATED is consistent with strategy from previous standards cahnge ( see HUD_RURAL_HOUSING_STABILITY_ASSISTANCE_PROGRAM_DEPRECATED)

gigxz replied:
ok, this is fine because the backing value is the integer, so 47 will now resolve as `HUD_ESG_CV_DEPRECATED` instead of `HUD_ESG_CV`. I think exception may raise if user is actively submitting a funder with the deprecated enum value, but refreshing the page would resolve that, so I think its OK.

gigxz replied:
Downside of this decision is that we need to maintain a frontend branch and keep deploying that too https://github.com/greenriver/hmis-frontend/pull/1251
Deploying it for now but we may want to back this out and keep the old enum, just to reduce overhead with maintaining and deploying both the branches.


Edit: I undid the breaking enum change here https://github.com/greenriver/hmis-warehouse/pull/5669/commits/f3518dfb91b111a52934811afab84f074ff27113. I know the approach was consistent with a previous standards change, but I think we made a mistake in that approach. Keeping keys consistent reduces the likelihood of user impact and makes it so that this branch doesn't break the API so we dont need to worry about deploying the frontend together

gigxz committed:
Merge branch 'release-178' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Remove component override

gigxz commented:
Converting to draft since this should not yet be merged into release. Deploying to staging 9/2

gigxz committed:
Merge branch 'release-178' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-178' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Dont render r13 in input group

gigxz committed:
Undo breaking API change for funder

gigxz committed:
Merge branch 'release-179' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-179' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-179' into 7853-hmis-hud-2026-breaking-changes

martha committed:
Merge branch 'release-179' into 7853-hmis-hud-2026-breaking-changes

martha committed:
Merge branch 'release-179' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-179' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-179' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-179' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-179' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-179' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-179' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-179' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-180' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-180' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-180' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-180' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-180' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-181' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-181' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-181' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-181' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-181' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-181' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-181' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-181' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-181' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-181' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-181' into 7853-hmis-hud-2026-breaking-changes

martha committed:
Merge branch 'release-182' into 7853-hmis-hud-2026-breaking-changes

ttoomey committed:
8234-fy26-changes (#5791)

- Removed VA rules from `disability_table` and `disability_table_r4` so elements 4.05–4.10 are not required for VA-funded projects
- Documentation for `hud_assessment_form_rules_2026.rb`.
- Set `"warn_if_empty" to false for: translation_assistance (C4) and Sexual Orientation (R3)

gigxz committed:
Merge branch 'release-182' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-182' into 7853-hmis-hud-2026-breaking-changes

gigxz committed:
Merge branch 'release-182' into 7853-hmis-hud-2026-breaking-changes

gigxz reviewed (approved):
Approving this for squash-merge into release-182 🙌
